### PR TITLE
Map color

### PR DESCRIPTION
**Pull request by slevin75**
This pr has new targets and modified map functions that I used to update the plots in the figures.

### CODE REVIEW

**Review by jds485 — COMMENTED** [review]
Thanks! I have some minor change comments and a suggestion for merging the function `make_class_prediction_map_panel_for_paper` with `make_class_prediction_map`. It would take some time to do that, so up to you to decide if that makes sense

File: 3_cluster/src/seasonal_metric_cluster.R [hunk 1]
```diff
@@ -725,3 +725,97 @@ get_colmeans_panel_plt <- function(metric_names, metric_mat, by_quantile,
...
+  #' 
+  #' @return filepath to the panel plot
+  #' 
+  
+  ##only making this plot using the following quantiles.
+  clust_table_cols <- c("ID","0.55_k5", "0.75_k5", "0.95_k5")
+  cluster_table <- cluster_table %>%
+    select(clust_table_cols)
+  
+  gages <- select(gages, 1:4, geometry) %>%
```
[the remark this answers is not in the document]
Comment (slevin75) [reply 3_cluster/src/seasonal_metric_cluster.R:750]: I think its the first 4 columns of the gages dataset.  This was copied from the plot_cluster_map function (starting on line 539). I can replace it with the column names if that would be better.

File: 6_predict/src/plot_diagnostics.R [hunk 2]
```diff
@@ -811,3 +811,102 @@ offset_partial <- function(partial_data){
   
   return(partial_data$yhat)
 }
+
+
+
```
[the remark this answers is not in the document]
Comment (jds485) [reply 6_predict/src/plot_diagnostics.R:816]: Okay, sounds good.

File: 3_cluster/src/seasonal_metric_cluster.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (jds485) [reply 3_cluster/src/seasonal_metric_cluster.R:750]: I think it is better for reproducibility to specify the column names. Could you change in both functions?

**Comment by jds485** [comment]
Let me know if you want help with the merge conflicts

**Comment by slevin75** [comment]
I have never had to deal with merge conflicts.  It looks like there are just some additional functions there that are not in my branch so I want to keep them both.  Do I just delete the <<<<< and ===== lines in the web editor?

**Comment by jds485** [comment]
Yes, delete <<<, ===, and >>> lines. Looks like in `_targets.R` there will be a package to retain, too.

**Pull request merged**